zeroclickinfo-spice/lib/DDG/Spice/Catfacts.pm

```diff
@@ -0,0 +1,21 @@
+package DDG::Spice::Catfacts;
```

Comment:
@RomainLG it looks like you've created this in a new sub-folder, `zeroclickinfo-spice` -- please move these files into the correct place and remove the `zeroclickinfo-spice` subdir you've created and then this will work as expected